docs/contribution.md

```diff
@@ -0,0 +1,80 @@
+# Contributing to the Databook
+
+The OpenScope Databook is a community project. It aims to serve as a collection of contributed analyses from many different experts and to include a comprehensive set of useful and reproducible analyses. Contributors are added to the authors list and have an opportunity to share their figures, results, and datasets through their notebooks in the Databook. Together the project can grow and continue to shape the growing reproducible neuroscience ecosystem.
+
+### Fork the OpenScope Databook
+The Databook can be forked via the Github Web UI from the Databook's [Github repo](https://github.com/AllenInstitute/openscope_databook). Press the [fork button](https://github.com/AllenInstitute/openscope_databook/fork) or click this link.
+
+### Initialize Locally
+A local repo can be made by pressing the `code` button on the front page of the forked repo, and copying the HTTPS url. Then locally, run the command `git init`, and then `git clone <copied_url_here>`. For more information on cloning Github repos, check out Github's [Cloning a Repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) Page.
+
+Then the environment must be set up. This can be done by navigating to the openscope_databook directory and running `pip install -e . --user`.
+
+Finally, notebooks can be run with Jupyter notebook by running `jupyter notebook ./docs`
+
+
+### Make Contributions
+A contribution can take the form of changes to an existing notebook. More likely, it will be in adding a new analysis notebook to the databook. First, determine which *chapter* it will go into within the `./docs` directory, and create a new `.ipynb` file there. Then the notebook can be produced with the analysis and reproducible figures. For a contribution to be accepted it must adhere to the contribution **Styleguide** below. In addition to having clean code and clear figures, it is important that the notebook has thorough markdown explaining the figures and the analysis performed.
+
+**Important Note:**
+For a notebook to be compliant with the Databook, it must have an environment setup cell at the top. This is described in the **Styleguide** below.
+
+Any additional packages required for the notebook to run should be added to `requirements.txt`. When the notebook is complete it must be added to `docs/_toc.yml` to the appropriate chapter
+
+
```

Comment:
Add: 
 ### How to make a review.
Review of the openscope databook are also made on Github via GIT commit. We recommend you make a local fork of this repository and create a PR with your edits that either correct or extend a given notebook. If you feel like a notebook should be discussed, please first make a PR request and enter your reviewing comment on the PR form on Github. You are free to choose any page of the databook for reviews, depending on your expertise and time. According to the authorships section below, reviewer gain authorship if they propose modifications that are eventually merged into the main branch.